### PR TITLE
OKD: Verify upgrades from latest on stream release channel

### PR DESCRIPTION
**Pull request by LorbusChris**
Verify that:
- New release on OKD/FCOS `4-stable` channel can be upgraded from current latest release on `4-stable`.
- New release on OKD/SCOS `4-scos-stable` channel can be upgraded from current latest release on `4-scos-stable`.
- New release on OKD/SCOS `4-scos-next` channel can be upgraded from current latest release on `4-scos-next`.

/cc @vrutkovs

### CODE REVIEW

**Comment by LorbusChris** [comment]
/pj-rehearse periodic-ci-openshift-release-master-okd-scos-4.14-e2e-aws-upgrade

**Comment by vrutkovs** [comment]
This seems to fix https://github.com/okd-project/okd/issues/1696

File: ci-operator/config/openshift/release/openshift-release-master__okd-4.13.yaml [hunk 1]
```diff
@@ -43,14 +50,32 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-aws-ovn
```
Comment (vrutkovs) [inline ci-operator/config/openshift/release/openshift-release-master__okd-4.13.yaml:46]: Lets leave `ovn` prefix - this is a requirement from Sippy folks to distinguish SDN/OVN jobs

File: ci-operator/config/openshift/release/openshift-release-master__okd-4.14.yaml [hunk 2]
```diff
@@ -32,6 +32,13 @@ base_images:
     namespace: origin
     tag: upi-installer
 releases:
+  initial:
+    prerelease:
+      product: okd
+      version_bounds:
+        lower: 4.13.0-0
```
Comment (vrutkovs) [inline ci-operator/config/openshift/release/openshift-release-master__okd-4.14.yaml:39]:
```suggestion
        lower: 4.14.0-0
```

File: ci-operator/config/openshift/release/openshift-release-master__okd-4.15.yaml [hunk 3]
```diff
@@ -32,6 +32,13 @@ base_images:
     namespace: origin
     tag: upi-installer
 releases:
+  initial:
+    prerelease:
+      product: okd
+      version_bounds:
+        lower: 4.13.0-0
```
Comment (vrutkovs) [inline ci-operator/config/openshift/release/openshift-release-master__okd-4.15.yaml:39]:
```suggestion
        lower: 4.14.0-0
```

File: ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.15.yaml [hunk 4]
```diff
@@ -3,8 +3,9 @@ releases:
     prerelease:
       product: okd
       version_bounds:
-        lower: 4.12.0
-        upper: 4.1000.0
+        lower: 4.13.0-0
```
Comment (vrutkovs) [inline ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.15.yaml:6]:
```suggestion
        lower: 4.14.0-0
```

Not sure if this picks 4.14 without this change

File: ci-operator/config/openshift/release/openshift-release-master__okd-4.15.yaml (hunk 3 above)
Comment (LorbusChris) [reply ci-operator/config/openshift/release/openshift-release-master__okd-4.15.yaml:39]: This shouldn't be necessary, as it'll always pick the latest one from https://origin-release.ci.openshift.org/#4-stable now

File: ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.15.yaml (hunk 4 above)
Comment (LorbusChris) [reply ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.15.yaml:6]: see https://github.com/openshift/release/pull/42843#discussion_r1311260404

File: ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.14.yaml [hunk 5]
```diff
@@ -17,20 +18,35 @@ resources:
...
+  interval: 48h
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    env:
+      TEST_TYPE: upgrade-conformance
+    observers:
+      enable:
+      - observers-resource-watch
+    workflow: openshift-upgrade-gcp
+- as: e2e-vsphere-ocp
```
Comment (vrutkovs) [inline ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.14.yaml:45]:
```suggestion
- as: e2e-vsphere-ovn
```

**Review by vrutkovs — APPROVED** [review]
/approve
/lgtm

**Comment by vrutkovs** [comment]
This needs another `make jobs` iiuc

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**